packages/beacon-node/src/api/impl/beacon/state/utils.ts

```diff
@@ -83,6 +83,28 @@ export async function getStateResponseWithRegen(
   return res;
 }
 
+export type GeneralValidatorStatus = "active" | "pending" | "exited" | "withdrawal";
```

Comment:
no need to export this type

```suggestion
type GeneralValidatorStatus = "active" | "pending" | "exited" | "withdrawal";
```